setup.cfg

```diff
@@ -0,0 +1,36 @@
+[metadata]
+name = git-machete
+summary = Probably the sharpest git repository organizer & rebase workflow automation tool you've ever seen ;)
+description-file = README.md
+license = MIT License
+classifiers =
+    Programming Language :: Python :: 2
+    Programming Language :: Python
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+keywords = git
+author = VirtusLab
+author-email = pawel.p.lipski@gmail.com
+home-page = https://github.com/VirtusLab/git-machete/
+project-url = https://github.com/VirtusLab/git-machete/
+
+[files]
+packages =
+    git_machete
+data_files =
+    /etc/bash_completion.d = extras/git-machete-prompt
```

Comment:
So `setup.py` still requires a `sudo`, right? Even in case of running as `python setup.py install --user`?

Reply:
It will be not needed. I updated it in https://github.com/VirtusLab/git-machete/pull/13/commits/b2fa5a66c1dc322352196a55c89fd87762e82100.